interface.js

```diff
@@ -0,0 +1,36 @@
+const storyHandler = new StoryHandler();
+var homePage;
+
+window.addEventListener("DOMContentLoaded",() => {
+  storyHandler.renderHeadlines()
+  .then(addClickListeners)
+  .then(storeHomePage)
+  addBackButtonListener();
+})
+
+function addBackButtonListener() {
+  let listButton = document.getElementById('listButton');
+  listButton.addEventListener('click', () => {
+    restoreHomePage();
+  });
+}
+
+function addClickListeners() {
+  let headlines = document.getElementsByClassName('headline')
+  for (let i = 0; i < headlines.length; i++) {
+    const headline = headlines[i];
+    headline.addEventListener('click', (e) => {
+      storyHandler.renderSummary(e.currentTarget.id)
+    })
+  }
+}
```

Comment:
I have changed my mind, I like the adding event listeners here because it's something I couldn't figure out how to do last week, but is there a way you could refactor this to have just a single event liistener for a click? that way you could have 1000s of articles on a page without having 1000s of event listeners